Dockerfile

```diff
@@ -1,15 +1,71 @@
-FROM php:8.1-fpm-alpine
+FROM ubuntu:22.04
```

Comment:
We should make `Dockerfile` for development purpose, so the review can play around with the container. No need to build production container image at this context.
With this developer image, you can do everything from inside container as you do in your local machine.